### ISSUE TITLE
Table 1  the DiT model refer to?

### ISSUE THREAD

**Issue by sillytheif** [vimar-gu/MinimaxDiffusion#20]
I have a question regarding the DiT method presented in your Table 1. Which of the following situations does the DiT here refer to?
 (a) A directly pre-trained DiT model
 (b)A pre-trained DiT model that was fine-tuned on ImageWoof using only the MSE loss, without the minimax constraint.

**Comment by vimar-gu** [comment]
Hi, it refers to (a) a direct DiT model pre-trained on ImageNet.
